vdev/src/testing/integration.rs

```diff
@@ -171,6 +197,7 @@ impl Compose {
 
     fn stop(&self) -> Result<()> {
         // The config settings are not needed when stopping a compose setup.
+        std::fs::remove_file(&self.path)?;
```

Comment:
Would like to know if this ultimately will be when we remove the temp file or if this should be handled elsewhere.

Reply:
I don't think the temporary file needs to stay present when the service is not being managed, as we can recreate it each time we run vdev, so I think we can let it be removed (i.e. with `TempFile`) each time we run.